src/main/flight/pid.c

```diff
@@ -664,15 +667,22 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         thrustLinearizationB = (1.0f - thrustLinearization) / (2.0f * thrustLinearization);
     }
 #endif    
-#if defined(USE_D_CUT)
-    dtermCutPercent = pidProfile->dterm_cut_percent / 100.0f;
-    dtermCutPercentInv = 1.0f - dtermCutPercent;
-    dtermCutRangeHz = pidProfile->dterm_cut_range_hz;
-    dtermCutLowpassHz = pidProfile->dterm_cut_lowpass_hz;
-    dtermCutGain = pidProfile->dterm_cut_gain * dtermCutPercent * D_CUT_GAIN_FACTOR / dtermCutLowpassHz;
+#if defined(USE_D_MIN)
+    dMin[FD_ROLL] = (pidProfile->d_min_roll);
+    dMin[FD_PITCH] = (pidProfile->d_min_pitch);
+    dMin[FD_YAW] = 0;
+    for (int axis = FD_ROLL; axis <= FD_YAW; ++axis) {
+        if ((dMin[axis] > 0) && (dMin[axis] < pidProfile->pid[axis].D)) {
+            dMinPercent[axis] = dMin[axis] / (float)(pidProfile->pid[FD_ROLL].D);
+        } else {
+            dMinPercent[axis] = 0;
+        }
+    }
+    dMinGyroGain = pidProfile->d_min_gain * D_MIN_GAIN_FACTOR / D_MIN_LOWPASS_HZ;
+    dMinSetpointGain = pidProfile->d_min_gain * D_MIN_SETPOINT_GAIN_FACTOR * pidProfile->d_min_advance * pidFrequency / (100 * D_MIN_LOWPASS_HZ);
     // lowpass included inversely in gain since stronger lowpass decreases peak effect
+    dMinFactor = 1.0f;
```

Comment:
This should be moved down into the `pidController` function - no need for it to be a static.

Reply:
Yes, thanks for explaining how; done.